### ISSUE TITLE
Enable accessibility support bundling QT accessible plugin while building Mixxx

### ISSUE THREAD

**Issue by Holzhaus** [Holzhaus/mixxx-gh-issue-migration#4009]
The QT4 framework Mixxx uses for gui controls provides basic accessibility support for the default QT vidgets. This is not built-in by default into the qt libraries. It can either be used as a library or QT can load it as a shared library during the runtime.
QT accessible plugin is currently available for Windows and for the mac.
Mixxx uses standard vidgets mainly in the preferences dialog, in the menu and also in the library. When this plugin is bundled Mixxx is at least partially accessible to visually disabled users.
For more complete accessibility support in Mixxx it's important to enhance gui control relation ships and provide other means of accessing features which are not accessible with the QT4 accessible plugin. However I think those are seprate issues and they should be described with proper examples and use cases as a seperate bug reports / wishes.
This is a required for visually disabled people to be able to get started with Mixxx and they might also be able to provide further wishes when this is available in the official releases.
